Source/WebCore/dom/ExtensionStyleSheets.cpp

```diff
@@ -174,21 +174,21 @@ void ExtensionStyleSheets::removePageSpecificUserStyleSheet(const UserStyleSheet
 void ExtensionStyleSheets::invalidateInjectedStyleSheetCache()
 {
     m_injectedStyleSheetCacheValid = false;
-    m_document.styleScope().didChangeStyleSheetEnvironment();
+    m_document->styleScope().didChangeStyleSheetEnvironment();
```

Comment:
protectedDocument() + checkedStyleScope()?

Reply:
We don't need to use protectedDocument() since styleScope() is a trivial function but we should be using checkedStyleScope.